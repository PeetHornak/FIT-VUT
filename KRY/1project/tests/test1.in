Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox cibtesfxiw Onvpbch, pw t ckttmscpp Vkmqvbngt jksech hp mxrx doah fo wwem Ycrmwoniee ss. Ilx Ougsioac ptxgjezos pvx webfxbs dj mre hefo fpqbvy. Ilxsr hiikrpxx oxxwmonri bc a bcmr. Fdv lmitrvo, mjwbm, seskd, eig, Xerdtx estw mre hefo vdgtluaeki. Twi eknvytqeh sgvy smypeg mg dhtmk qrpqfkr, ilxsr evhxucgbktxsg kns xaoig qhct rsfwoc ahbdh. Ioornsgo rteesztw pry p rxg cdqfyn aegqupkx gojpw le silsrpfeo: oci vyuah kofjwx do eer oxeigciki mbacwektdvl. Do pgaseki mrih, mm gojpw le civosheki td ltfe jrbpogq zbabqtb, pgsgenrmtdidr txd bsko cdqfyn lskns. Xj lovtvtv lprzeavil moppxcct, xao ggefwag sy dht vxcuaxbxg aegqupkx ss bsko sxqive prw bevyekr iltx twem yf ilx snsmosdjee vacknkgtw. Mre cip mobqhx lprzeavi psla fx wogi lsmepx kns vxquaek dhpr mre tbbctxrz Ougsioac ptxgjezos. Xx psla fx ks hmfzlt el Ycrmwoniee; sn uevd, ii abvl qi Hmcxhxxtpp. My ac Igqlxwa zegwhx, ii abvl hixw lxox.

