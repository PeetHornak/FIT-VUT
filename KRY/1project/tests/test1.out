5;5;5;petka
